test/history.cpp

```diff
@@ -34,6 +35,31 @@ TEST_CASE("History can be iterated on in any direction", "[History]")
 				REQUIRE(h.next() == "foobar");
 				REQUIRE(h.next() == "");
 				REQUIRE(h.next() == "");
+
+                                SECTION("Save and Load History to file")
```

Comment:
The test looks good, but I'd turn it into an actual `TEST_CASE` because:
1. short tests are easier to understand, and this `SECTION` only needs three lines from the previous sections;
2. the nesting here is a bit too much :)